client/state/purchases/test/index.js

```diff
@@ -0,0 +1,202 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import Dispatcher from 'dispatcher';
+
+/**
+ * Internal dependencies
+ */
+import {
+	PURCHASES_USER_FETCH,
+	PURCHASES_SITE_FETCH_COMPLETED,
+	PURCHASES_USER_FETCH_COMPLETED,
+	PRIVACY_PROTECTION_CANCEL_COMPLETED,
+	PRIVACY_PROTECTION_CANCEL_FAILED
+} from 'state/action-types';
+import { getByPurchaseId } from '../selectors';
+import { items } from '..';
+
+describe( 'items', () => {
+	const userId = 1337,
+		siteId = 2701;
+
+	it( 'should return an object with the initial state', () => {
+		expect( items( undefined, { type: 'UNRELATED' } ) ).to.be.eql( {
```

Comment:
I didn't consider doing this because I just copied the tests. @Tug has mentioned that he thinks this is a bad idea but I can't remember why.